### ISSUE TITLE
Suitable for children indication

### ISSUE THREAD

**Issue by SteveAmor** [south-west-communities/website#41]
Would it be possible to have an indicator/icon that shows if an event is suitable for children to attend?

**Comment by tonyedwardspz** [comment]
Good idea.

This would be for tech jam / exeter raspberry jam / code dojo only out of current organiser set?

**Comment by SteveAmor** [comment]
Open Tech Challange?
Data science Cornwall?
They may be 14+ (I don't know for sure) so it may be a little more complex than it first looks :-)

**Comment by tonyedwardspz** [comment]
I've looked into this and have decided to not implement this feature at this moment in time, but may come back to it in the future but in a simplistic way, e.g labeling only those that are explicitly kid-focused.

Essentially, there are too many caveats in making this work effectively. In particular, I don't feel comfortable in stating an event might be suitable for Children when it's not. There was also a question raised by the alpha testers as to whether an event suitable for 14+ should be labeled in the same way as one suitable for 8+. We then ended up down a rabbit hole of considering a film like classification (U, PG, 12......). Ultimately, the decision was to focus on other features.

Won't implement in the near future so have closed this issue.
